type/src/ai-card.ts

```diff
@@ -0,0 +1,197 @@
+// --8<-- [start:AICard]
+/**
+ * The unified AI Card object for a single agent.
+ */
+export interface AICard {
+  /**
+   * Declares the schema this card adheres to.
+   * e.g., "https://ai-agent-protocol.org/ai-card/v1/schema.json"
+   */
+  $schema: string;
+
+  /**
+   * The version of the AI Card specification itself.
+   */
+  specVersion: string;
+  
+  /**
+   * The primary verifiable ID for the agent (e.g., DID).
+   */
+  id: string; 
+  
+  /**
+   * A human-readable name for the agent.
+   */
+  name: string;
+
+  /**
+   * A short, human-readable description of the agent's purpose.
+   */
+  description: string;
+
+  /**
+   * A direct URL to the agent's logo image.
+   */
+  logoUrl?: string;
```

Comment:
Enterprises are likely not going to want to dereference URLs to retrieve this content.  It would be good to clarify that data URLs are supported here. https://www.rfc-editor.org/rfc/rfc2397